experiments/20161006-hackytaq-replicates/20161006-hackytaq-replicate.md

```diff
@@ -0,0 +1,95 @@
+## Goals
+
+Following up on Vivian's set of excellently-conducted prior experiments (date: [2016-09-20](../20160920-Taq-Mn-mutagenesis/20160912-mgcl2-taq-series.md)), I wanted to fill in the blanks w.r.t. the concentrations not yet tested and sequenced.
```

Comment:
"fill in the blanks" refers to testing the Manganese concentrations not sent for sequencing in your previous run ([note](https://github.com/ericmjl/protein-systematic-characterization/blob/master/experiments/20160920-Taq-Mn-mutagenesis/20160912-mgcl2-taq-series.md)). I will update this paragraph explicitly.